frontend/src/app/accountCreation/MfaSendCodeToContact/MfaSendCodeToContact.tsx

```diff
@@ -0,0 +1,135 @@
+import { useEffect, useState } from "react";
+import { Redirect } from "react-router";
+
+import { Card } from "../../commonComponents/Card/Card";
+import { CardBackground } from "../../commonComponents/CardBackground/CardBackground";
+import TextInput from "../../commonComponents/TextInput";
+import Button from "../../commonComponents/Button/Button";
+import StepIndicator from "../../commonComponents/StepIndicator";
+import { accountCreationSteps } from "../../../config/constants";
+import { phoneNumberIsValid } from "../../patients/personSchema";
+import { emailIsValid } from "../../utils/email";
+import { capitalizeText } from "../../utils/text";
+
+interface Props {
+  type: "phone number" | "email address";
+  serviceEnroll: Function;
+  cardText: string;
+  cardHint?: string;
+}
+
+export const MfaSendCodeToContact = (props: Props) => {
+  const [contact, setContact] = useState("");
+  const [contactError, setContactError] = useState("");
+  const [formIsDirty, setFormIsDirty] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+
+  const contactIsValid =
+    props.type === "phone number" ? phoneNumberIsValid : emailIsValid;
+
+  const validateContact = () => {
+    setFormIsDirty(true);
+    let error = "";
+    if (!contact) {
+      error = `Enter your ${props.type}`;
+      setContactError(error);
+      return false;
+    }
+
+    let valid;
+    try {
+      valid = contactIsValid(contact);
+    } catch (e) {
+      valid = false;
+    }
+
+    if (!valid) {
+      error = `Enter a valid ${props.type}`;
+    }
+    setContactError(error);
+    return error === "";
+  };
+
+  const handleSubmit = async () => {
+    if (validateContact()) {
+      setLoading(true);
+      try {
+        await props.serviceEnroll(contact);
+        setSubmitted(true);
+      } catch (error) {
+        setContactError(`API Error: ${error}`);
```

Comment:
This error is the entire response object, right? So will this contact error look like "API Error: [object Object]"?

Reply:
Yeah, good catch. I don't actually know what the error object will look like though. I suppose `error.message` is a good bet?